deploy/docker/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.8.7-slim
 
 # Update
 RUN apt update -y && apt upgrade -y
-RUN apt install patch gcc libffi-dev wget -y
+RUN apt install patch gcc libffi-dev wget git -y
```

Comment:
Could you do the same for `deploy/docker/porter/Dockerfile` as well?

Reply:
It's just a temporary measure, because I'm currently pulling `umbral` from a git commit. It will be removed when I set it to ==0.3

Reply:
Oh I see - 👍